go.mod

```diff
@@ -1,6 +1,6 @@
 module sigs.k8s.io/apiserver-network-proxy
 
-go 1.22.5
+go 1.22.7
```

Comment:
This file lost a "toolchain" directive fairly recently, should we restore it?

Reply:
I'd like to get it back here and in the matching test-infra repo. However right now I'd like to unblock dependabot.